src/main/java/frc/robot/subsystems/chassis/Elevator.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.chassis;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Elevator extends SubsystemBase {
+  private CANSparkMax ElevNeoMotor1;
+  private CANSparkMax ElevNeoMotor2;
```

Comment:
lowercase variable names. Makes the code more readable since `Classes` are usually capitalized.